src/Microsoft.AspNetCore.Razor.Language/Legacy/TagHelperBlockRewriter.cs

```diff
@@ -16,13 +16,14 @@ internal static class TagHelperBlockRewriter
         public static TagHelperBlockBuilder Rewrite(
             string tagName,
             bool validStructure,
+            bool allowMinimizedBooleanTagHelperAttributes,
```

Comment:
Have this take in the feature flags. Future TagHelper changes will be much easier to complete.